loco-gen/src/templates/scaffold/html/base.t

```diff
@@ -16,15 +16,56 @@ message: "Base template was added successfully."
   {% raw %}{% endblock head %}{% endraw %}
 </head>
 
-<body class="prose p-10">
-  <div id="content">
-    {% raw %}{% block content %}
-    {% endblock content %}{% endraw %}
-  </div>
-
+<body class="min-h-screen bg-background font-sans antialiased">
+    <div class="relative flex min-h-screen flex-col bg-background">
+        <div class="themes-wrapper bg-background">
+            <div style="--sidebar-width:16rem;--sidebar-width-icon:3rem" class="group/sidebar-wrapper flex min-h-svh w-full has-[[data-variant=inset]]:bg-sidebar">
+                <main class="relative flex min-h-svh flex-1 flex-col bg-background peer-data-[variant=inset]:min-h-[calc(100svh-theme(spacing.4))] md:peer-data-[variant=inset]:m-2 md:peer-data-[state=collapsed]:peer-data-[variant=inset]:ml-2 md:peer-data-[variant=inset]:ml-0 md:peer-data-[variant=inset]:rounded-xl md:peer-data-[variant=inset]:shadow">
```

Comment:
in the context of a "fresh" scaffold file, tailwind classes make it a very unhappy place to edit
can we create custom classes in our own CSS (or maybe in base) and use a single class in the different view files?

Reply:
done